### DISCUSSION TITLE
install/install-tools/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1220] [Announcements]
# install/install-tools/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/install-tools/

**Comment by zgfh** [comment]
临时包： https://qiniu-download-public.daocloud.io/opensource/tmp-/data/dce-5.0/install-tools/install-tools-0.6.0.tgz-1da3afd224fb155bcc351ace26c7123d

**Reply by jzhupup** [reply]
arm 架构的临时包。

**Comment by bill-ding007** [comment]
离线安装商业版应该是：./install_prerequisite_${VERSION}.sh offline full

**Comment by leolingg** [comment]
对于商业版：
```
export BINARY_TAR=prerequisite_${VERSION}_amd64.tar.gz
chmod +x install_prerequisite_${VERSION}.sh
./install_prerequisite_${VERSION}.sh offline community
```
`./install_prerequisite_${VERSION}.sh offline community` 是部署需要改成 `./install_prerequisite_${VERSION}.sh offline full`

**Comment by Ye-Ting** [comment]
VERSION=v0.9.0  需更新到 v0.10.0

**Reply by KubeKyrie** [reply]
依赖项还包括 charts-syncer

**Reply by jzhupup** [reply]
已添加

**Comment by san3Xian** [comment]
希望prerequisite同样能在下载中心中有独立页面维护下载地址

**Reply by windsonsea** [reply]
下载中心已增加了前置依赖包的下载：
https://docs.daocloud.io/download/#_1

**Reply by lgp2013** [reply]
这个页面一直不更新了，最新的是export VERSION=v0.27.0

**Reply by windsonsea** [reply]
Fixed in #6341 and updated all inline installer versons to v0.27.0
